### ISSUE TITLE
port message uuid migration to standalone script for 0.4.x instances

### ISSUE THREAD

**Issue by mandric** [medic/medic-webapp#3054]
Related: 
https://github.com/medic/medic-api/pull/116
https://github.com/medic/medic-webapp/issues/3015
https://github.com/medic/medic-webapp/issues/3033

**Comment by mandric** [comment]
assigning to @garethbowen for comment, see branch/PR https://github.com/medic/medic-api/pull/119

**Comment by garethbowen** [comment]
Looks good. Might have issues with lots of messages but I suspect our target instances won't be that big (ie: somewhat less than LG...).

I prefer to script in node because it's installed on the servers and we're all experts in it, but that's a personal preference, especially in this case as this is only needed in the short term.

**Comment by garethbowen** [comment]
Back to you @mandric

**Comment by sglangevin** [comment]
Do we have an issue for applying this script to the instances that need it? Or does this issue cover that piece as well?

**Comment by mandric** [comment]
Good point. I don't have an issue for applying the fix to instances that have messages without uuids.  Unless someone feels strongly about it I'll incorporate it into this ticket.

**Comment by mandric** [comment]
Nevermind, tracking this over here now: https://github.com/medic/medic-projects/issues/1168.  This branch/PR has gone through a refactor though and should get another look.  Reassigning to @garethbowen to review.

**Comment by garethbowen** [comment]
Looks ok. Shame about using http vs request but I can see why you did. I wonder if we could have the script run an npm install for dependencies (uuid, request, etc)? It's fine for now.

**Comment by mandric** [comment]
Pretty sure I'm only one that can do A/T here.... so closing this.
